go/test/endtoend/reparent/reparent_test.go

```diff
@@ -79,6 +78,10 @@ func TestReparentDownMaster(t *testing.T) {
 		// Reset status, don't wait for the tablet status. We will check it later
 		tablet.VttabletProcess.ServingStatus = ""
 
+		// Init Tablet
+		err = clusterInstance.VtctlclientProcess.InitTablet(&tablet, tablet.Cell, keyspaceName, hostname, shardName)
+		require.NoError(t, err)
+
```

Comment:
@sougou should all the calls to InitTablet be removed? And if yes, what other changes are needed?

Reply:
Oh yes, these need to be removed, they are deprecated.

Reply:
What do I replace them with?

Reply:
Just delete them. vttablet startup now includes whatever InitTablet used to do.

Reply:
Removing them worked out just fine.